cap-ci/README.md

```diff
@@ -24,16 +33,19 @@ E.g: to deploy the `cap-release` pipeline:
 
     $ ./create_pipeline.sh <concourse-target> cap-release
 
-Note: If you wish to deploy a custom pipeline, copy and modify either `cap-pre-release.yaml` or `cap-release.yaml`. The name of your config file will be used as the `<pipeline-name>`.
+Note: If you wish to deploy a custom pipeline, copy and modify either
+`cap-pre-release.yaml` or `cap-release.yaml`. The name of your config file will
+be used as the `<pipeline-name>`.
 
 # Implementation
 
-The pipeline strives to have the minimum concourse yaml to do the job, and put
+The pipeline strives to have the minimum concourse YAML to do the job, and put
 the logic somewhere so one can run and iterate on the pipeline locally.
-It's still in flux.
 
-# K8s cluster management
+# Kubernetes cluster management
 
-The pipeline consumes clusters by putting a lock on unclaimed kubeconfigs from git@github.com:SUSE/cf-ci-pools.git
+The pipeline consumes clusters by putting a lock on unclaimed kubeconfigs from
+git@github.com:SUSE/cf-ci-pools.git.
 
-Add your kubeconfigs to cf-ci-pools.git in ${Backend}-kube-hosts branch in the unclaimed folder for CI to pick it up.
+Add your kubeconfigs to cf-ci-pools.git in the `${Backend}-kube-hosts` branch
+under the `unclaimed` folder so CI can pick them up.
```

Comment:
or enable `deploy-k8s` which will create a new kubernetes cluster for selected public cloud and add its kubeconfig to cf-ci-pools. Note:  `deploy-k8s` only works for public platform at this point of time.